airbyte-integrations/connectors/source-trello/pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "1.0.3"
+name = "source-trello"
+description = "Source implementation for Trello."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/trello"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
```

```suggestion
[tool.poetry.packages]
```
Not sure why we have double brackets there, was this autogenerated by the migration command?